text-generation-inference/docker/Dockerfile

```diff
@@ -84,6 +106,7 @@ RUN VERBOSE=1 BUILDDIR=/pyserver/build PROTODIR=/pyserver/proto VERSION=${VERSIO
 FROM base AS tpu_base
 
 ARG VERSION=${VERSION}
+RUN test -n ${VERSION:?}
```

Comment:
you added version at line 70, now we have it here again, do we need both?

Reply:
No we don't need it indeed